src/ifElse.test.js

```diff
@@ -1,11 +1,32 @@
+/* eslint-disable no-console */
 'use strict';
 
 describe('ifElse', () => {
-  // const { ifElse } = require('./ifElse');
+  const { ifElse } = require('./ifElse');
 
-  it('should ', () => {
+  it(`should call 'f' if first argument is 'true'`, () => {
+    const f = jest.fn();
+    const cb = () => console.log(2);
 
+    ifElse(() => true, f, cb);
+
+    expect(f).toHaveBeenCalled();
+  });
+
+  it(`should call 'f' if first argument is 'false'`, () => {
+    const f = jest.fn();
+    const cb = () => console.log(1);
+
+    ifElse(() => false, cb, f);
+
+    expect(f).toHaveBeenCalled();
```

Comment:
This expectation is incorrect. Since the condition is false, the 'cb' function should be called, not 'f'. Update the expectation to `expect(cb).toHaveBeenCalled();`.